demos/simpleBuildAgainstTrilinos/CMakeLists.txt

```diff
@@ -54,17 +54,19 @@ endif()
 
 # Build the APP and link to Trilinos
 add_executable(MyApp ${CMAKE_CURRENT_SOURCE_DIR}/app.cpp)
-target_include_directories(MyApp PRIVATE
-  ${CMAKE_CURRENT_SOURCE_DIR} ${Trilinos_INCLUDE_DIRS} ${Trilinos_TPL_INCLUDE_DIRS})
-target_link_libraries(MyApp ${Trilinos_LIBRARIES} ${Trilinos_TPL_LIBRARIES}) 
+target_link_libraries(MyApp  Trilinos::all_selected_libs)
+# Or, above could have linked to just Tpetra::all_libs
```

Comment:
Is `set(CMAKE_CXX_FLAGS  "${Trilinos_CXX_COMPILER_FLAGS} ${CMAKE_CXX_FLAGS}")` as left alone above supposed to still be needed? Shouldn't necessary compiler flags be exposed by the interface inherited from Trilinos via `target_link_libraries`?

Reply:
@PhilMiller, that is up to the downstream CMake project.  Really compiler options coming from Kokkos IMPORTED targets in the INTERFACE_COMPILE_OPTIONS property should be automatically propagated as well as other usage requirements coming like the required language standard.